### PR TITLE
Provide option to check if path exists

### PR DESCRIPTION
**Pull request by michielspiritus**

### CODE REVIEW

**Comment by royteeuwen** [comment]
@andresbott  could you provide a review please?

File: src/main/java/com/adobe/cq/testing/junit/assertion/CQAssert.java [hunk 1]
```diff
@@ -86,6 +98,36 @@ public Boolean call() {
...
+                public Boolean call() {
+                    try {
+                        return client.exists(path);
+                    } catch(Throwable e) {
+                        return false;
+                    }
+                }
+            }.poll(timeout, delay);
+        } catch (TimeoutException e) {
+            Assert.fail("Timeout reached while waiting for path to be deleted: " + path);
```
Comment (bisswanger) [inline src/main/java/com/adobe/cq/testing/junit/assertion/CQAssert.java]: I guess the error message here does not apply for the new function?
Rather something like

```suggestion
            Assert.fail("Timeout reached while waiting for path to be created: " + path);
```

**Review by bisswanger — APPROVED** [review]

**Review by dulvac — APPROVED** [review]
lgtm

**Comment by bisswanger** [comment]
@michielspiritus: Thanks for the contribution.
I merged the changes and released version 1.2.2

I released it via a staging repository to Maven Central.
It should be available there in the next few hours:
https://mvnrepository.com/artifact/com.adobe.cq/aem-cloud-testing-clients

**Pull request merged**